src/main/osd/osd_elements.c

```diff
@@ -1065,8 +1065,8 @@ static void osdElementWarnings(osdElementParms_t *element)
        gpsRescueIsDisabled()) {
 
         statistic_t *stats = osdGetStats();
-        if (!cmpTimeUs(stats->armed_time, OSD_GPS_RESCUE_DISABLED_WARNING_DURATION_US)) {
-            osdFormatMessage(element->buff, OSD_FORMAT_MESSAGE_BUFFER_SIZE, "RESC OFF!!!");
+        if (cmpTimeUs(stats->armed_time, OSD_GPS_RESCUE_DISABLED_WARNING_DURATION_US) < 0) {
+            osdFormatMessage(element->buff, OSD_FORMAT_MESSAGE_BUFFER_SIZE, "\x1E\x1F RESC OFF"); //0x1E=SYM_SAT_L, 0x1F=SYM_SAT_R
```

Comment:
We actually have definitions for those symbols:

https://github.com/betaflight/betaflight/blob/80b3a116d3ff16a348874426edc782f486b2ca69/src/main/drivers/max7456_symbols.h#L65-L66

And we should be using the symbols, as this will also mean we do not accidentally consider the characters to be unused and remove them by pulling the symbols and reusing the slots.

Reply:
I knew about those definitions, what I don't know is how to concatenate constant characters and strings, such as 'a' and "b" at compile time. The only way I know (with runtime code) looks a bit overkill for such a simple case. I put the definitions on the comments to alleviate any confusion reading the code. So, what's preferred?

Reply:
```
char message[] = "   RESC OFF";
message[0] = SYM_SAT_L;
message[1] = SYM_SAT_R;
osdFormatMessage(element->buff, OSD_FORMAT_MESSAGE_BUFFER_SIZE, message);
```

Reply:
To me is a little strange use the image of the sat to show the warning, I prefer the old way only text, but this is only my preference.

Reply:
I was trying to avoid "GPSRESC OFF", which is less legible imho. Maybe just "RESCUE OFF" could be fine. This could also match with "RESCUE N/A".

Reply:
On second thought, @McGiverGim is right, and using a symbol in the warning text probably isn't a good idea - just try to imagine users wanting to open an issue around this, and not being able to replicate the warning they are getting...

"RESCUE OFF" / "RESCUE N/A" sounds good to me.

Reply:
Me too. @TonyBlit can you change the messages, please?

Reply:
I haven't thought about this O:-) totally agreed! Do you mind if I also change the N/A message in this PR, or is better to do it in a new PR?

Reply:
Go for both, @TonyBlit, they are part of the same feature.